tests/Doctrine/Tests/Common/DataFixtures/TestEntity/Link.php

```diff
@@ -32,17 +32,17 @@ public function __construct(Uuid $id)
         $this->id = $id;
     }
 
-    public function getId()
+    public function getId(): Uuid
     {
         return $this->id;
     }
 
-    public function getUrl()
+    public function getUrl(): string
```

Comment:
`url` could actually be `null` because it's not initialized in the constructor.

But this is just a test fixture so I'm not sure how important this is.